### PR TITLE
convert ConversationListCell

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

convert `ConversationListCell` to Swift

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListCell.swift [hunk 1]
```diff
@@ -85,7 +173,54 @@ extension ConversationListCell {
...
+    }
+    
+    
+    override func setVisualDrawerOffset(_ visualDrawerOffset: CGFloat, updateUI doUpdate: Bool) {
+        super.setVisualDrawerOffset(visualDrawerOffset, updateUI: doUpdate)
+        
+        // After X % of reveal we consider animation should be finished
+        let progress = visualDrawerOffset / MaxVisualDrawerOffsetRevealDistance
+        menuDotsView.setProgress(progress, animated: true)
+        if progress >= 1 && overscrollStartDate != nil {
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListCell.swift]: Comparing with `ConversationListCell.m`:

```objective-c
if (progress >= 1 && ! self.overscrolStartDate) {
  self.overscrollStateDate = [NSDate date];
}
```

Looks like you actually want to check the `overscrollStartDate == nil`.

File: Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ConversationListContentController.swift [hunk 2]
```diff
@@ -270,6 +270,7 @@ final class ConversationListContentController: UICollectionViewController {
             fatal("Unknown cell type")
         }
 
+        ///TODO: compile error?
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ConversationListContentController.swift]: Todo.

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListContentController/ConversationListContentController.swift]: fixed after cleaning.

File: Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListCell.swift (hunk 1 above)
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/ConversationList/ListContent/ConversationListCell.swift]: fixed!

**Pull request merged**